packages/ts-type-expand/src/vsc/type-expand-provider.ts

```diff
@@ -16,6 +16,7 @@ export class TypeExpandProvider
     type: TypeObject
   }
   private activeFilePath: string | undefined
+  private topElement?: ExpandableTypeItem
```

Comment:
[want] 欲しいのは表示済みかの状態だけなので、boolean で十分で意図が読みやすいかなと思いました！Element 自体持たせている意図が特になければこちらでお願いしたいです！

```suggestion
  private isAlreadyShowTopElement: boolean = false
```